apps/dapp-console/app/settings/contracts/page.tsx

```diff
@@ -6,8 +6,17 @@ import { ClaimedRebateProgressBanner } from '@/app/settings/components/ClaimedRe
 import { CoinbaseVerificationBanner } from '@/app/settings/components/CoinbaseVerificationBanner'
 import { DeployedApps } from '@/app/settings/contracts/DeployedApps'
 import { ClaimedRebates } from '@/app/settings/components/ClaimedRebates'
+import { apiClient } from '@/app/helpers/apiClient'
+import { useEffect } from 'react'
 
 export default function Contracts() {
+  const { mutate: syncCbVerification } =
```

Comment:
Adding this sync so it always happens on the contract page, so people can get unstuck with a reload. I think we should look into sharing the cb verified code so the FE can start polling this when we have the rebate dialog modal open. We can poll EAS from the FE and when we find an attestation we call this sync from the FE.